packages/wallet-sdk/src/util/web.ts

```diff
@@ -1,8 +1,8 @@
-import { NAME, VERSION } from '../sdk-info.js';
-import { getCrossOriginOpenerPolicy } from './checkCrossOriginOpenerPolicy.js';
 import { standardErrors } from ':core/error/errors.js';
-import { Snackbar } from ':sign/walletlink/relay/ui/components/Snackbar/Snackbar.js';
 import { RETRY_SVG_PATH } from ':sign/walletlink/relay/ui/WalletLinkRelayUI.js';
+import { Snackbar } from ':sign/walletlink/relay/ui/components/Snackbar/Snackbar.js';
+import { NAME, VERSION } from '../sdk-info.js';
+import { getCrossOriginOpenerPolicy } from './checkCrossOriginOpenerPolicy.js';
```

Comment:
is this included on accident?

Reply:
When I ran lint this was picked up. Since it was only an import change I kept it